### DISCUSSION TITLE
Windows Compatability

### DISCUSSION THREAD

**Discussion by 26mmangan** [holly-hacker/BulletForceHaxV2#18] [Q&A] [answered]
Can you use this on windows? If so, how? I don't see anything about this in the README file.

**Comment by holly-hacker** [comment, marked as answer]
Yes, I do my main development on Windows. You can find a pre-built executable in the releases and the README contains info on how to enable hax. Right now there aren't any useful cheats, though.

I have a branch with a new web-based UI but it's not finished yet and I have little motivation to work on it. If/when that is merged, I'll start adding some cheats.
